src/components/style/interactive/button.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  --button-blue: 0, 102, 153;
+  --button-red: 229, 50, 62;
+}
+
+.fdsButton {
+  border: 0 none;
+  padding: var(--space-half) var(--space-default);
+  border-radius: var(--border-radius-default);
+  font-weight: 500;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-size: var(--fontSize--m);
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  line-height: normal;
+  transition: background-color var(--button-group-timeout) var(--motion-easing-default),
+    box-shadow var(--button-group-timeout) var(--motion-easing-default),;
+}
+
+
+.fdsButton--loading,
+.fdsButton--disabled {
+  opacity: 0.6;
+  pointer-events: none;
+}
+
+@keyframes spinner {
+  to {transform: rotate(360deg);}
+}
+
+/* Spinner technique is from here: https://stephanwagner.me/only-css-loading-spinner */
+.fdsButton--loading::before {
+  content: "";
+  box-sizing: border-box;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 16px;
+  height: 16px;
+  margin-top: -8px;
+  margin-left: -8px;
```

Comment:
I'm not "sure" I like the idea of using variables here, given that these values always need to be half of the width and height of this element (so if the variables ever change in the future, this loader would break.... (#resilientCSS))